examples/demo/demo.ts

```diff
@@ -18,7 +18,10 @@ import { OrderedListButton } from '../../packages/plugin-list/src/ListButtons';
 import { UnorderedListButton } from '../../packages/plugin-list/src/ListButtons';
 import { IndentButton } from '../../packages/plugin-indent/src/IndentButtons';
 import { OutdentButton } from '../../packages/plugin-indent/src/IndentButtons';
-import { AlignLeftButton, AlignCenterButton, AlignRightButton, AlignJustifyButton } from '../../packages/plugin-align/src/AlignButtons';
+import { AlignLeftButton } from '../../packages/plugin-align/src/AlignButtons';
+import { AlignCenterButton } from '../../packages/plugin-align/src/AlignButtons';
+import { AlignRightButton } from '../../packages/plugin-align/src/AlignButtons';
+import { AlignJustifyButton } from '../../packages/plugin-align/src/AlignButtons';
```

Comment:
What was the issue? I wasn't aware it was not allowed to have several imports on the same line.

Reply:
It is, but the line was too long and my linter complained about it so I broke it into multiple lines to fix it.